packages/babel-parser/src/tokenizer/index.js

```diff
@@ -1033,11 +1033,7 @@ export default class Tokenizer extends LocationParser {
       this.input.charCodeAt(start) === charCodes.digit0;
     if (octal) {
       if (this.state.strict) {
-        this.raise(
-          start,
-          // todo: merge with Errors.StrictOctalLiteral
-          "Legacy octal literals are not allowed in strict mode",
-        );
+        this.raise(start, Errors.StrictOctalLiteral);
```

Comment:
Here we might want to change the `StrictOctalLiteral` message, because `0o1` is perfectly valid in strict mode.

Reply:
Yeah, `0o1` is octal literal but not _legacy_ octal literals. I have changed `StrictOctalLiteral` to be "Legacy octal literals are not allowed in strict mode".